content/rs/concepts/shard-placement-policy.md

```diff
@@ -40,7 +40,7 @@ For example, for a database with two master and two slave shards on a cluster wi
 - Node 3 hosts the second master shard
 - Node 1 hosts for the slave shard for master shard 2
 
-The dense policy helps to use fewer system resources.
+The sparse policy spreads the data across more nodes to improve network utilization.
```

Comment:
On the contrary: sparse causes more network traffic.
It is useful for RoF bdbs, where the storage becomes a bottleneck.